pkg/operator/config/crd/bases/mps.playfab.com_gameserverbuilds.yaml

```diff
@@ -383,10 +383,73 @@ spec:
                                             only "value". The requirements are ANDed.
                                           type: object
                                       type: object
+                                    namespaceSelector:
```

Comment:
curious why these fields were added. Did we change controller-runtime version?

Reply:
These must have snuck in when I was testing with kubebuilder locally, I'll revert the changes.